src/hotspot/share/gc/shenandoah/shenandoahBarrierSet.inline.hpp

```diff
@@ -272,8 +276,7 @@ inline void ShenandoahBarrierSet::AccessBarrier<decorators, BarrierSetT>::oop_st
 
   oop_store_common(addr, value);
   if (ShenandoahCardBarrier) {
-    ShenandoahBarrierSet* bs = ShenandoahBarrierSet::barrier_set();
-    bs->write_ref_field_post<decorators>(addr);
+    barrier_set()->write_ref_field_post<decorators>(addr);
```

Comment:
Unnecessary change?

Reply:
Yes, just idly fixing warnings in my editor. I'll revert it.